src/js/components/aboutTheDataSidebar/AboutTheDataListView.jsx

```diff
@@ -16,6 +17,11 @@ const AboutTheDataListView = ({ section, selectItem }) => {
     const clickHandler = (e, index, section) => {
         e.preventDefault();
         selectItem(index, section);
+        Analytics.event({
+            category: 'About the Data',
+            action: `Clicked ${section}`,
+            label: 'About the Data'
```

Comment:
the label here isn't helpful.  i think it could be removed or it could be formatted like About the Data - <section name> - <term name>